### PR TITLE
Support Julia 0.5.0

### PR DESCRIPTION
**Pull request by wookay**
Hello, I'd like to submit the pull request for Julia 0.5.0

Here's an example on heroku.
http://bukdu.herokuapp.com/

### CODE REVIEW

**Comment by pinx** [comment]
Looks good! I guess you don't know of a way to get the julia hash? Seems a bit awkward that I have to specify it in the code.
Bukdu seems like exactly what I need! Currently, I am calling Julia via Porcelain from Phoenix.

**Comment by wookay** [comment]
thanks! regard about awkward hash, have submit the new pull request.
Phoenix is great. but elixir :)

File: bin/compile [hunk 1]
```diff
@@ -47,11 +49,29 @@ tar xzf $JULIA_BINARIES -C $BUILD_DIR
...
+CMAKE_BINARIES="$CACHE_DIR/cmake.tar.gz"
+if [ -f $CMAKE_BINARIES ]; then
+  echo "Using CMake binaries from cache" | indent
+else
+  echo "Downloading binaries ($CMAKE_URL)" | indent
+  curl -L $CMAKE_URL -o $CMAKE_BINARIES 
+fi
+
+echo "Unpacking binaries" | indent
+tar xzf $CMAKE_BINARIES -C /tmp/codon/vendor
```
Comment (pinx) [inline bin/compile]: I got an error message, saying the directory doesn't exist. I fixed it by adding mkdir -p

File: bin/compile [hunk 2]
```diff
@@ -47,11 +49,29 @@ tar xzf $JULIA_BINARIES -C $BUILD_DIR
...
+
+cmake --version
```
Comment (pinx) [inline bin/compile]: I got an error here, saying that cmake could not be found. I removed it for now. Do we need cmake???

Comment (pinx) [inline bin/compile]: If we don't need cmake, can we remove it alltogether?

Comment (wookay) [inline bin/compile]: `MbedTLS.jl` has required cmake to build.
MbedTLS is required by HttpServer.

```
INFO: Building MbedTLS
cmake command not found. installing cmake is required for building from source.
===============================[ ERROR: MbedTLS ]===============================
LoadError: failed process: Process(`./cmake_check.sh`, ProcessExited(1)) [1]
while loading /app/.julia/v0.5/MbedTLS/deps/build.jl, in expression starting on line 74
================================================================================
================================[ BUILD ERRORS ]================================
WARNING: MbedTLS had build errors.
 - packages with build errors remain installed in /app/.julia/v0.5
 - build the package(s) and all dependencies with `Pkg.build("MbedTLS")`
 - build a single package by running its `deps/build.jl` script
```

Comment (pinx) [inline bin/compile]: cmake is apparently installed in /tmp/codon/vendor/bin. This is not in my path, so just calling cmake will fail.
I looked into the elixir buildpack. I think we need to do something similar, i.e. put stuff in platform_tools_path and add that to $PATH.
https://github.com/HashNuke/heroku-buildpack-elixir/blob/master/lib/path_funcs.sh

Comment (wookay) [inline bin/compile]: submit new PR for this. removed cmake and adding another buildpack for cmake.
https://github.com/pinx/heroku-buildpack-julia/pull/4
thanks.

**Pull request merged**